lib/web_ui/lib/src/engine/compositor/util.dart

```diff
@@ -152,7 +171,8 @@ js.JsArray<double> makeSkPoint(ui.Offset point) {
 Float32List encodePointList(List<ui.Offset> points) {
   assert(points != null);
   final int pointCount = points.length;
-  final Float32List result = canvasKit.callMethod('Malloc', <dynamic>[js.context['Float32Array'], pointCount * 2]);
+  final Float32List result = canvasKit.callMethod(
+      'Malloc', <dynamic>[js.context['Float32Array'], pointCount * 2]);
```

Comment:
Not in 0.15.0, because everywhere the resulting list is passed (SkCanvas.drawPoints, SkCanvas.drawAtlas, SkPath.addPoly) it is automatically freed by CanvasKit. However, in the next version it will switch to not free-ing Malloc'ed arrays. I will add a TODO.